vertx-sql-client/src/main/java/io/vertx/sqlclient/SqlResult.java

```diff
@@ -57,6 +57,8 @@
    */
   int size();
 
+  <V> V property(PropertyKind<V> property);
```

Comment:
Missing javadoc

Reply:
added in https://github.com/eclipse-vertx/vertx-sql-client/pull/392/commits/3ba7628e94292eecbf57f94a0179a6af15204215.